frontend/src/libs/components/popover/popover.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState } from '~/libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  content: React.ReactNode;
+  trigger: React.ReactNode;
+};
+
+const Popover: React.FC<Properties> = ({
+  content,
+  trigger,
+}: Properties): JSX.Element => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const handleClick = useCallback((): void => {
+    setIsOpen(!isOpen);
+  }, [isOpen]);
+
+  const handleKeyDown = useCallback(
+    (event: React.KeyboardEvent<HTMLDivElement>) => {
+      if (event.key === 'Enter') {
+        setIsOpen(!isOpen);
+      }
+    },
+    [isOpen],
+  );
+
+  return (
+    <div className={styles['popover-wrapper']}>
+      <div
+        role="button"
+        tabIndex={0}
+        onClick={handleClick}
+        onKeyDown={handleKeyDown}
```

Comment:
Not sure if we really need it

Reply:
i had error from eslint like this:

https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-static-element-interactions.md

Reply:
@kuslosolodka good. left it for it!

Reply:
I'm about `onKeyDown` event handler

Reply:
'Visible, non-interactive elements with click handlers must have at least one keyboard listener'

Reply:
@andanf-e we have a linter for it
https://www.npmjs.com/package/eslint-plugin-jsx-a11y